tests/core/scene/a-scene.test.js

```diff
@@ -28,6 +28,24 @@ suite('a-scene (without renderer)', function () {
     test('initializes scene object', function () {
       assert.equal(this.el.object3D.type, 'Scene');
     });
+
+    test('does not initialize systems', function (done) {
+      var sceneEl = document.createElement('a-scene');
+      process.nextTick(function () {
```

Comment:
In the conversation section of the PR I see the nextTick still but If I go to changes it's not there anymore. ![nextTick](https://s3.amazonaws.com/f.cl.ly/items/0M0f1G2T3Z1x0c2t082k/Image%202016-07-26%20at%208.32.47%20PM.png?v=5b849c13)